amip_interp/include/amip_interp_r4.fh

```diff
@@ -0,0 +1,71 @@
+#undef AMIP_INTERP_KIND_
```

Comment:
FMS_AMIP_INTERP_KIND_? :)

Reply:
Renamed in a15e58ea3fb062be330ce3e29219d7f1b6f959fa.